src/ini.cr

```diff
@@ -19,4 +19,21 @@ class INI
     end
     ini
   end
+
+  # Generates an INI-style configuration from a given hash.
+  #
+  # ```
+  # INI.build({"foo" => {"a" => "1"}}, " ") # => "[foo]\na = 1\n\n"
+  # ```
+  def self.build(ini : Hash(String, Hash(String, String)), space : String = "") : String
+    io = IO::Memory.new
+    ini.each do |key, value|
+      io << "[" << key << "]\n"
+      value.each do |str|
```

Comment:
Let's do `.each do |key, value|` here and rename the existing `key` to `section` and the existing `value` to `contents`.